src/main/resources/org/allenai/pipeline/pipelineSummary.html

```diff
@@ -337,13 +337,13 @@
          *
          * @return {string}   The HTML for the step contents.
          */
-        function generateStepContent(label, description, millis, data, links) {
+        function generateStepContent(label, description, millis, source, data, links) {
           var out = "<h2>" + label + "</h2>";
           if(millis) {
-            out += millis + " millis";
+            out += millis + " millis" + " (" + source + ")";
```

Comment:
Since run time could be hours, maybe formatting the runtime as a Duration (ms, s, h) is better.